### ISSUE TITLE
Accessibility issue on the expand/collapse (should be listed as headings)

### ISSUE THREAD

**Issue by lmichaud** [hc-sc/rsa-renewal#7]
Current WET library has expand/collapse as regular containers (not headers) breaking accessibility. Need to raise as an issue on main WET repo.
